compiler/rustc_mir_build/src/check_unsafety.rs

```diff
@@ -909,11 +909,6 @@ impl UnsafeOpKind {
 }
 
 pub fn check_unsafety(tcx: TyCtxt<'_>, def: LocalDefId) {
```

Comment:
in separate PR: should this get moved into it's own crate `rustc_thir_unsafeck` or `rustc_thir_analysis`?